specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -328,6 +328,7 @@ function RecordSet<SCHEMA extends AnySchema>({
           )
         }
         ids={ids}
+        totalCount={totalCount}
```

Comment:
this line can probably be removed as `ids` length should always equal `totalCount`